### ISSUE TITLE
Update database if youtube content is updated? (e.g. title, description)

### ISSUE THREAD

**Issue by narrowmm** [mxpv/podsync#219]
As title.   Sometimes the youtuber updates the title of their video, I wonder if it is possible that Podsync can detect that and help updating the database?

Thanks.

**Comment by chudeyang0225** [comment]
Agreed. I'm following this uploader who does live and then edits the video title after the live. Can't just skip the 'live' by the filter because the downloader will ignore this same video even after the title is changed.

**Comment by mxpv** [comment]
The main challenge with implementing this is that it would require checking every video for metadata changes on each sync, which would quickly deplete a user's API quota.
